### PR TITLE
Party

### PR DESCRIPTION
**Pull request by zacksingh**
Implement partySize drop-down selector using sampleData

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      date: '',
+      time: '7:00 pm',
+      partySize: 2
+    }
+
+    this.partySizes = ['1 person'];
```
Comment (ColeHuntley83) [inline client/src/components/App.jsx]: is the array defaulted to show 1 person? and then when genratePartSizes gets called , it builds up the array of party size choices with 'one person' on the front.. just understanding your code thats all.

File: database/sampleData.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const sampleData =
```
Comment (ColeHuntley83) [inline database/sampleData.js]: i like this. keep it simple... no need to use data from API yet.

File: client/src/components/App.jsx (hunk 1 above)
Comment (zacksingh) [reply client/src/components/App.jsx]: That's exactly how it works! The database tracks one bit of data related to party size, and that's an integer that represents the maximum party size. So on the client side, when it gets that number, it just builds an array of all the party size options up to that max, to be used in the selector.

File: package.json [hunk 3]
```diff
@@ -5,22 +5,44 @@
...
+    "webpack-cli": "^3.3.10"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --config ./jest.config.json",
+    "test-watch": "npm run test -- --watch",
     "seed": "node ./database/seeding.js",
-    "start": "nodemon ./server/index.js"
+    "start": "nodemon ./server/index.js",
+    "react-dev": "webpack -d --watch"
```
Comment (ColeHuntley83) [inline package.json]: what does the flag '-d'
 do?

File: client/src/components/App.jsx (hunk 1 above)
Comment (ColeHuntley83) [reply client/src/components/App.jsx]: ahh nice. makes sense. very elegant

**Pull request merged**